app/admin/documents.rb

```diff
@@ -35,9 +35,45 @@
     actions
   end
 
+  csv do
+    column :id
+    column :name
+    column :external_url
+    column :language
+    column :last_verified_on
+    column 'Law ID', &:document_legacy_law_id
```

Comment:
I'm not sure if we need that legacy law ID for our import/export.

Reply:
This is the only way to match documents with legislations. This could be handy during: `export -> manual change -> import` scenario. Also, it doesn't break anything.

I do agree that in the future, after the initial import, this probably will be removed.

Reply:
That's documentable_id for.

Reply:
ok, I removed this for now